### ISSUE TITLE
sandbox generation fails with running seeds

### ISSUE THREAD

**Issue by peterberkenbosch** [solidusio/solidus_dev_support#89]
```
Loading seed file: refund_reasons
Loading seed file: roles
Loading seed file: shipping_categories
rake aborted!
NameError: undefined local variable or method `spree_api_key' for #<Spree::User:0x00007fb55e4911a8>
Did you mean?  spree_role_ids
```

**Comment by skukx** [comment]
Bump:

```
Loading seed file: shipping_categories
rake aborted!
NameError: undefined local variable or method `spree_api_key' for #<Spree::User:0x00007f9b09f74508>
Did you mean?  spree_role_ids
/Users/taylorscott/.rvm/gems/ruby-2.6.5@solidus_kts/gems/activemodel-6.0.2.1/lib/active_model/attribute_methods.rb:431:in `method_missing'
/Users/taylorscott/.rvm/gems/ruby-2.6.5@solidus_kts/gems/solidus_auth_devise-2.4.0/app/models/spree/user.rb:15:in `password='
```

**Comment by peterberkenbosch** [comment]
Thank @skukx, This is fixed in #90, my expectation is that this will be merged tomorrow.
